Dockerfile

```diff
@@ -12,48 +12,78 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi8/ubi
+FROM registry.access.redhat.com/ubi8/openjdk-8
+
+# Build parameters
+ARG IQ_SERVER_VERSION=1.99.0-01
+ARG IQ_SERVER_SHA256=b8e13545503bf4b8a93a7918fc6a56e330a620c1533f91cd9ffcb4bbaaee23c5
+ARG TEMP="/tmp/work"
+ARG IQ_HOME="/opt/sonatype/nexus-iq-server"
+ARG SONATYPE_WORK="/sonatype-work"
+ARG CONFIG_HOME="/etc/nexus-iq-server"
+ARG LOGS_HOME="/var/log/nexus-iq-server"
+
+ENV DOCKER_TYPE="docker"
 
 LABEL vendor=Sonatype \
   maintainer="Sonatype <support@sonatype.com>" \
   com.sonatype.license="Apache License, Version 2.0" \
   com.sonatype.name="Nexus IQ Server image"
 
-# Optional parameters.
-ARG IQ_SERVER_VERSION=1.99.0-01
-ARG IQ_SERVER_SHA256=b8e13545503bf4b8a93a7918fc6a56e330a620c1533f91cd9ffcb4bbaaee23c5
+USER root
+
+# For testing
+RUN microdnf update \
+&& microdnf install procps
 
-# Mandatory parameters. Docker needs to know volume mount point and location of startup script.
-ENV SONATYPE_WORK="/sonatype-work" \
-    IQ_HOME="/opt/sonatype/nexus-iq-server/" \
-    DOCKER_TYPE="docker"
-
-ARG IQ_SERVER_COOKBOOK_VERSION="release-0.4.20200928-152944.e90ad8b"
-ARG IQ_SERVER_COOKBOOK_URL="https://github.com/sonatype/chef-nexus-iq-server/releases/download/${IQ_SERVER_COOKBOOK_VERSION}/chef-nexus-iq-server.tar.gz"
-
-ADD solo.json.erb /var/chef/solo.json.erb
-
-# Install using chef-solo
-RUN yum install -y --disableplugin=subscription-manager hostname procps \
-    && curl -L https://www.getchef.com/chef/install.sh | bash -s -- -v 14.12.9 \
-    && /opt/chef/embedded/bin/erb /var/chef/solo.json.erb > /var/chef/solo.json \
-    && chef-solo \
-       --recipe-url ${IQ_SERVER_COOKBOOK_URL} \
-       --json-attributes /var/chef/solo.json \
-    && rpm -qa *chef* | xargs rpm -e \
-    && rm -rf /etc/chef \
-    && rm -rf /opt/chefdk \
-    && rm -rf /var/cache/yum \
-    && rm -rf /var/chef \
-    && yum clean all
+# Create folders
+RUN mkdir -p ${TEMP} \
+&& mkdir -m 0755 -p ${IQ_HOME} \
+&& mkdir -m 0755 -p ${SONATYPE_WORK} \
+&& mkdir -m 0755 -p ${CONFIG_HOME} \
+&& mkdir -m 0755 -p ${LOGS_HOME}
 
+# Copy config.yml and set sonatypeWork to the correct value
+COPY config.yml ${TEMP}
```

Comment:
To my knowledge: From where is this config.yml placed here (in to the running directory)?

Reply:
> To my knowledge: From where is this config.yml placed here (in to the running directory)?

Ah, ignore ☝️ . I can see you have added it below in the root.

Reply:
> To my knowledge: From where is this config.yml placed here (in to the running directory)?

The `config.yml` is being added in this PR, in the same location as the `Dockerfile` so as long as the running directory is where the `Dockerfile` is this should be correct